sympy/core/tests/test_basic.py

```diff
@@ -336,6 +337,12 @@ class B(A[T]):
 
 
 def test_rewrite_abs():
-    from sympy import Piecewise, Eq
+    """
+    Test that rewrite(abs) behaves identically to rewrite(Abs).
+
+    Related to issue 27327: https://github.com/sympy/sympy/issues/27327
+    Ensure rewrite(abs) calls rewrite(Abs) and produces expected results.
+    """
```

Comment:
Hmm I guess a link to the issue is enough :)

Reply:
Should I remove the comments?

Reply:
I mean yeah, a link to the issue should be enough.

Reply:
Okay I will do that.